.gitignore

```diff
@@ -34,12 +34,4 @@ out/
 /.nb-gradle/
 
 ### VS Code ###
-.vscode/**
-!.vscode/launch.json
-!.vscode/settings.json
-
-### Durable Prom store
-infrastructure/prometheus/durable
-
-### Durable Loki store
-infrastructure/loki/durable/
```

Comment:
We should add these ignored files back so that they don't make our lives more difficult when we end up implementing for example Prometheus or Loki tooling